lucid-book-store/src/components/cart-wishlist/cart.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { CartCard } from "./cart-card";
+import { CartSummary } from "./cart-summary";
+import "./cart.css";
+
+const Cart = () => {
+  return (
+    <>
```

Comment:
Is it necessary to use two react fragments.

Reply:
That was a typo, fixed it thanks